.gitignore

```diff
@@ -24,6 +24,7 @@ parts/
 sdist/
 var/
 wheels/
+.idea/
```

Comment:
As I'm using ``PyCharm`` and the IDE creates this folder to hold their configuration files there, I added it to be excluded from ``git`` as well

Reply:
That's fine!